### PR TITLE
added version check

### PR DESCRIPTION
**Pull request by thinkberg**
still need to fix the fixtures

### CODE REVIEW

**Comment by thinkberg** [comment]
please bump the version :-)

File: lib/zuck/facebook/ad_creative.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module Zuck
   class AdCreative < RawFbObject
 
     # Can't create this directly (yet)
-    read_only
+    # read_only
```
Comment (jayniz) [inline lib/zuck/facebook/ad_creative.rb]: Let's remove this all together if this functionality has been implemented!

File: lib/zuck/facebook/ad_group.rb [hunk 2]
```diff
@@ -7,18 +7,18 @@ class AdGroup < RawFbObject
...
                :conversion_specs,
                :campaign_id,
                :campaign_group_id,
-               :conversion_specs,
                :created_time,
                :creative_ids,
                :failed_delivery_checks,
                :name,
-               :targeting,
+               # :targeting,
```
Comment (jayniz) [inline lib/zuck/facebook/ad_group.rb]: Also not available to send, but in a response?

File: lib/zuck/facebook/targeting_spec.rb [hunk 3]
```diff
@@ -154,7 +154,8 @@ def self.fetch_reach(graph, ad_account, options)
 
     def validate_spec
       @spec[:interests] = @spec[:interests]
-      @spec[:broad_age] ||= false
+      # deprecated: https://developers.facebook.com/docs/apps/migrations/migration-2014-01-08
+      # @spec[:broad_age] ||= false
```
Comment (jayniz) [inline lib/zuck/facebook/targeting_spec.rb]: Then out with these lines :-)

File: spec/vcr_setup.rb [hunk 4]
```diff
@@ -10,6 +10,6 @@
     record: :new_episodes,
     match_requests_on: [:method, :body, :uri]
   }
-  c.allow_http_connections_when_no_cassette = false
+  c.allow_http_connections_when_no_cassette = true
```
Comment (jayniz) [inline spec/vcr_setup.rb]: Forgot to set this back to false :)

**Pull request merged**